app/src/main/java/cm/aptoide/pt/AptoideApplication.kt

```diff
@@ -78,4 +92,27 @@ class AptoideApplication : Application() {
       Timber.plant(Timber.DebugTree())
     }
   }
+
+  private fun calculateMd5Sum() {
+    aptoideMd5Manager.calculateMd5Sum()
+  }
+
+  private fun setUpAptoideUuid() {
+    generateAptoideUuid().andThen(
+      Completable.mergeDelayError(setUpInitialAdsUserProperty()),
+    )
+  }
+
+  private fun generateAptoideUuid(): Completable {
```

Comment:
Do we need Rx Java ? We should try and use kotlin flows

Reply:
i've removed RxJava, now working with Java flows and datastores